charts/pega/templates/_pega-eks-ingress.tpl

```diff
@@ -37,6 +37,13 @@ metadata:
     # enable sticky sessions on target group and set alb routing algorithm
     alb.ingress.kubernetes.io/target-group-attributes: load_balancing.algorithm.type=least_outstanding_requests,stickiness.enabled=true,stickiness.lb_cookie.duration_seconds={{ include "lbSessionCookieStickiness" . }}
 {{- end }}
+# protocol will be set to https only when either ingress is enabled or domain is set
+{{- if or (.node.service.domain) (.node.ingress) }}
```

Comment:
Could not understand what is to deal with `.node.service.domain` for setting tls? Same with AKS & GKE

Reply:
will make the changes on these